recipe/meta.yaml

```diff
@@ -1,34 +1,30 @@
 {% set name = "tslearn" %}
-{% set version = "0.5.3.2" %}
-{% set sha256 = "acf4015b07207ad6a645ddcc8eec04d8431716b24db66a745ab29fa22f772820" %}
+{% set version = "0.6.3" %}
 
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: c855c0b784b42dc4c50f4eaf524ec7ed658037e86fc14af38ca3b66c75dd6adb
 
 build:
   number: 0
-  skip: true  # [py2k]
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: true  # [py>312 or s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv --no-build-isolation
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - cython                                 # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
-    - {{ compiler('c') }}
+    - python
+    - numpy
```

Comment:
The build section is redundant because there are no compilers and build tools

Reply:
removing

Reply:
The build is still here...